docs/source/conf.py

```diff
@@ -51,4 +44,13 @@
 exclude_patterns = ['Thumbs.db', '.DS_Store']
 
 html_theme = 'sphinx_rtd_theme'
-pygments_style = 'friendly'
\ No newline at end of file
+pygments_style = 'friendly'
+
+
+# -- Allow documentation building without loading libraries -------------------
+
+os.environ['CDDS_NO_IMPORT_LIBS'] = "1"
```

Comment:
Can we change this to `CYCLONEDDS_NO_IMPORT_LIBS` to avoid confusion? I know it's nitpicking, it was already in there, but I prefer using the actual project name consistently...